packages/terra-form-fieldset/src/Fieldset.jsx

```diff
@@ -52,7 +57,7 @@ const Fieldset = ({
 
   return (
     <fieldset {...customProps} className={fieldsetClasses}>
-      {legend && <legend {...legendAttrs} className={legendClasses}>{legend}</legend>}
+      {legend && <legend {...legendAttrs} className={cx([legendClasses, { 'legend-visually-hidden': isLegendHidden }])}>{legend}</legend>}
```

Comment:
Lets move the legend-visually-hidden class to the legendClasses defined above on lines 53-56

Reply:
done 1a18a510b19e29aef23f0ae7303c2a7a390042d2